prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -369,7 +369,7 @@ periodics:
       - "bash"
       args:
       - "-c"
-      - "development/clusters-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=false -clusterNameRegexp='^gke(int|-upgrade|-central)[-](pr|commit|rel)[-].*'"
+      - "development/clusters-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=false -strategy=time"
```

Comment:
With the change in strategy I would suggest to set dryRun mode for a day, for verification, if we are removing what we intend to.

Reply:
added